docs/components/iconography/icon-detail-platform-tabs.tsx

```
@@ -0,0 +1,129 @@
+    <div className="flex flex-col gap-3">
+      <div className="flex items-center justify-between gap-2">
+        <span className="text-sm font-mono font-medium truncate">{pascalCaseName}</span>
+        <CopyButton text={pascalCaseName} label="컴포넌트 이름이 복사되었습니다" />
+      </div>
+      <DynamicCodeBlock lang="tsx" code={usageCode} />
+    </div>
+  );
+};
+
+const FigmaTab = () => {
+  const { selectedIcon, iconStyle } = useIcon();
```

Biome (2.4.4)

[error] This variable iconStyle is unused.

(lint/correctness/noUnusedVariables)